core/controllers/collection_editor.py

```diff
@@ -197,8 +219,33 @@ def put(self, collection_id):
 
         # TODO(bhenning): Implement other rights changes here.
         is_public = self.payload.get('is_public')
-
-        if is_public is not None:
+        is_publicized = self.payload.get('is_publicized')
```

Comment:
Let's drop this. The concept has been deprecated for explorations (we now manage the 'featured' list by a different mechanism) and I don't think we need to introduce it for collections.